vividus-tests/src/main/resources/story/integration/JbehaveCore.story

```diff
@@ -0,0 +1,5 @@
+Description: Integration tests for JBehave core.
```

Comment:
`Description: Integration tests for composite steps`

Reply:
fixed